### ISSUE TITLE
ESRI WKT on Python 3.5

### ISSUE THREAD

**Issue by gregory-halverson** [karimbahgat/PyCRS#11]
I'm getting `TypeError: string argument without an encoding` when I try to produce ESRI WKT on python 3.

```python
import pycrs
from pycrs.parser import from_proj4

crs = from_proj4('+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs ')
crs.to_esri_wkt()
```

    ---------------------------------------------------------------------------

    TypeError                                 Traceback (most recent call last)

    <ipython-input-3-cb542042cd0b> in <module>()
          3 
          4 crs = from_proj4('+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs ')
    ----> 5 crs.to_esri_wkt()
    

    C:\Users\Gregory\Anaconda2\envs\python3\lib\site-packages\pycrs\elements\containers.py in to_esri_wkt(self)
         21 
         22     def to_esri_wkt(self):
    ---> 23         return "%s" % self.toplevel.to_esri_wkt()
         24 
         25 ##+proj      Projection name (see `proj -l`)
    

    C:\Users\Gregory\Anaconda2\envs\python3\lib\site-packages\pycrs\elements\containers.py in to_esri_wkt(self)
        156 
        157     def to_esri_wkt(self):
    --> 158         return 'GEOGCS["%s", %s, %s, %s, AXIS["Lon", %s], AXIS["Lat", %s]]' % (self.name, self.datum.to_esri_wkt(), self.prime_mer.to_esri_wkt(), self.angunit.to_esri_wkt(), self.twin_ax[0].esri_wkt, self.twin_ax[1].esri_wkt )
        159 
        160 #PROJCS
    

    C:\Users\Gregory\Anaconda2\envs\python3\lib\site-packages\pycrs\elements\parameters.py in to_esri_wkt(self)
        364 
        365     def to_esri_wkt(self):
    --> 366         return 'UNIT["%s", %s]' %(self.unittype.to_esri_wkt(), self.metermultiplier.to_esri_wkt())
        367 
        368 ##+x_0       False easting
    

    C:\Users\Gregory\Anaconda2\envs\python3\lib\site-packages\pycrs\elements\parameters.py in to_esri_wkt(self)
        323 
        324     def to_esri_wkt(self):
    --> 325         return bytes(self.value.esri_wkt)
        326 
        327 # special...
    

    TypeError: string argument without an encoding

```python

```

Works fine in python 2, though.

```python
import pycrs
from pycrs.parser import from_proj4

crs = from_proj4('+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs ')
crs.to_esri_wkt()
```

    'GEOGCS["Unknown", DATUM["D_WGS_1984", SPHEROID["WGS_1984", 6378137, 298.257223563]], PRIMEM["Greenwich", 0], UNIT["Degree", 0.0174532925199], AXIS["Lon", EAST], AXIS["Lat", NORTH]]'

```python

```

**Comment by gregory-halverson** [comment]
I tried replacing all instances of `bytes` with `str` and got it to work in python 3.

https://github.com/gregory-halverson/PyCRS

**Comment by karimbahgat** [comment]
That's great, thanks for spotting that. I also tested that your changes on Python 2. Could you submit it as a PR? I will accept it and merge into the main branch.

**Comment by karimbahgat** [comment]
Merged, see #12.
